src/components/hero-decorative/hero-decorative.js

```diff
@@ -0,0 +1,49 @@
+import './hero-decorative.scss'
+import template from './hero-decorative.pug'
+import heroData from '../../data/main-nav.json'
+
+export default class HeroDecorative {
+  constructor (selector) {
+    this.node = document.querySelector(selector)
+    this.node.innerHTML = template(heroData)
+    this.imageContainer = this.node.querySelector('.hero2__img')
+    this.setDefaultImage()
+    this.mediaQueries()
+  }
+
+  static get sizes () {
+    return {
+      small: `url('${heroData.header2.images.small.url}')`,
+      medium: `url('${heroData.header2.images.medium.url}')`,
+      large: `url('${heroData.header2.images.large.url}')`
+    }
+  }
+
+  setDefaultImage () {
+    if (window.innerWidth < 640) {
```

Comment:
The `640` should not be hardcoded here. You should abstract this number into a reusable functionality / module.